docker-for-mac/release-notes.md

```diff
@@ -11,6 +11,8 @@ toc_max: 2
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+By downloading Docker Desktop, you agree to the terms of the [Docker Software End User License Agreement](https://www.docker.com/legal/docker-software-end-user-license-agreement){: target="_blank" rel="noopener" class="_"} and the [Docker Data Processing Agreement](https://www.docker.com/legal/data-processing-agreement){: target="_blank" rel="noopener" class="_"}.
```

Comment:
Should we create an include for this?

Reply:
Good idea :) I'll create an include and update the files

Reply:
Done. PTAL